### PR TITLE
node specifics added to travis and Readme

### PR DESCRIPTION
**Pull request by udnisap**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,10 @@
-language: objective-c
+language: node_js
 
 cache:
     directories:
         - node_modules
         - .nvm
 
-install:
-  - brew reinstall nvm
```
Comment (vjeux) [inline .travis.yml]: Thanks! I had no idea what I was doing, I just copy and pasted that from react native

File: README.md [hunk 2]
```diff
@@ -86,3 +86,16 @@ If you want to run the bot on your GitHub Enterprise instance, add the GHE host
 
 ## License
 mention-bot is BSD-licensed. We also provide an additional patent grant.
+
+
+[npm-url]: https://npmjs.org/package/mention-bot
```
Comment (vjeux) [inline README.md]: Please get rid of all those, they aren't useful

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# mention-bot [![Build Status](https://travis-ci.org/facebook/mention-bot.svg?branch=master)](https://travis-ci.org/facebook/mention-bot)
+# mention-bot [![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Build status][appveyor-image]][appveyor-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
```
Comment (vjeux) [inline README.md]: Please get rid of everything but mention bot

**Comment by vjeux** [comment]
Yay, thanks for cleaning up after me :)

**Comment by vjeux** [comment]
Can you sign the CLA? I can't merge it without :(

File: README.md (hunk 2 above)
Comment (udnisap) [inline README.md]: should this be released as a npm package?

**Comment by udnisap** [comment]
I have signed the CLA. I used a different email address and as a result it shows this. I changed the commit author email. Now it should be ok.

File: README.md (hunk 2 above)
Comment (vjeux) [inline README.md]: probably not? I'm not really sure what you would use a npm package for

**Comment by vjeux** [comment]
Yay it's green, thanks a lot!

screen shot 2015-11-22 at 8 34 25 pm

**Comment by udnisap** [comment]
Its a nice project. I found out in the react-native and is really cool. 
Thanks for making it open source

**Comment by vjeux** [comment]
You're welcome :) I initially wrote it for react-native as I was trying to figure out ways to make us review more pull requests and it's been working extremely well for us. I've been regularly asked how non-fb projects could use it so I ended up porting it to javascript (from php/hack) and open sourcing it over the weekend :p

**Pull request merged**